docs/gathered-data.md

```diff
@@ -399,6 +399,17 @@ Relevant OpenShift API docs:
 Location in archive: config/securitycontentconstraint/, config/clusterrolebinding/
 
 
+## SAPLicenseManagementLogs
+
+collects logs from license management pods with the following substrings:
```

Comment:
Please mention SAP here as well...I think it should be as explicit as possible in case of these SAP enhancements.

Reply:
fixed